feat(crons): Add environment support to the HTTP examples (#6634)
getsentry/sentry-docs: src/platforms/common/crons/index.mdx

</Note>

### Specifying monitor environments (Optional)

When sending check-ins to your monitor you may specify the `environment` of the
check-in. This allows you to monitor a single schedule across multiple
environments.

> If you do not specify an environment with your check-ins the default is `production`.

Reviewer: ```suggestion
If you don't specify an environment with your check-ins the default is `production`.
```

<Alert level="info">

> Monitor environments are still early in development. Currently after a check-in

Reviewer: ```suggestion
Monitor environments are still early in development. Currently, after a check-in
```
